src/Files.App/Views/MainPage.xaml

```diff
@@ -119,67 +119,50 @@
 
 	<Grid>
 		<Grid.RowDefinitions>
+			<RowDefinition Height="Auto" />
```

Comment:
Is this intentional?

Reply:
Yes. this is kind of refactoring; i removed grid that wrapped tab view and address toolbar which is unnecessary.